modules/tinymce/Gruntfile.js

```diff
@@ -175,6 +179,37 @@ module.exports = function (grunt) {
             }
           ]
         };
+      }),
+      gruntUtils.generate(models, 'model', (name) => {
+        return {
+          options: {
+            treeshake: true,
+            format: 'iife',
+            onwarn: swag.onwarn,
+            plugins: [
+              swag.nodeResolve({
+                basedir: __dirname,
+                prefixes: gruntUtils.prefixes({
+                  'tinymce/core': 'lib/globals/tinymce/core',
+                  'tinymce/ui': 'lib/ui/main/ts'
```

Comment:
This doesn't look right? That API/module was remove in 5.x and definitely isn't something a model should be using.
```suggestion
```

Reply:
Looks like the theme spec still has this as well, so it likely never was cleaned up when the `ui` directory was removed in 5.0 😞

Reply:
ouch. I guess good thing these changes highlighted that!